trino-aws-proxy-spi/src/main/java/io/trino/aws/proxy/spi/credentials/Identity.java

```diff
@@ -0,0 +1,18 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.aws.proxy.spi.credentials;
+
+public interface Identity
```

Comment:
Empty marker interfaces aren't ideal. Are there any standard methods we can add? `String user()`?

Reply:
It would be helpful to have a test with some custom variants of `Identity` to see how it might be used in a real case.